src/Build/BackEnd/Shared/BuildResult.cs

```diff
@@ -380,6 +381,12 @@ public ProjectInstance ProjectStateAfterBuild
             set => _projectStateAfterBuild = value;
         }
 
+        /// <summary>
+        /// Gets or sets the flags that provide additional control over the build request.
+        /// See <see cref="Execution.BuildRequestDataFlags"/> for examples of the available flags.
+        /// </summary>
+        public BuildRequestDataFlags BuildRequestDataFlags { get; }
```

Comment:
This will need to have an explicit field and the field should be handled in the `Translate` method.